### DISCUSSION TITLE
Newly created organization and repo doesn't show up in Visual Studio 2017 with GitHub plugin.

### DISCUSSION THREAD

**Discussion by GlitterHorn** [community/community#22614] [New to GitHub]
<p>I’ve created a new organization to host a repository for a project. My account is already a member of another organization.</p>
<p>In Visual Studio, I can only clone repositories from my personal account and the other organization - the organization I created doesn’t show up. I got around this by using Git desktop to clone the repository and then open it in VS, THEN sync it remotely, but shouldn’t I be able to see the organization directly in the GitHub plugin?</p>
<p>Thanks!</p>

**Comment by meaghanlewis** [comment, marked as answer]
<p>Hi <a class="mention" href="/clairelyclaire">@clairelyclaire</a> I’m guessing that you can’t see the organization you created because of permissions. Go to your organization then navigate to <strong>Settings</strong> -&gt; <strong>Third-party access</strong>. What is the default that you see for the access policy? You might need to remove restrictions to get the organization to appear in Visual Studio.</p>
<p>This article also explains more about OAuth App access restrictions: <a href="https://help.github.com/articles/about-oauth-app-access-restrictions/" rel="nofollow noopener">https://help.github.com/articles/about-oauth-app-access-restrictions/</a></p>
<p>Let me know how that works.</p>
<p>Thanks,</p>
<p>Meaghan</p>

**Comment by GlitterHorn** [comment]
<p>That fixed it. Thanks so much!</p>

**Comment by GlitterHorn** [comment]
<p>So I was just documenting this (setting up an org for access from VS), and now I’m not able to see a newly-created organization in Visual Studio via the GitHub plugin, even though I’ve removed application restrictions - the policy for the org shows “No restrictions”.</p>
<p>I still don’t see the organization or its repository in Visual Studio, though.</p>
<p>Okay, NOW it’s showing up. What delay should a user expect when setting this up for the first time?</p>
